samples/nrf9160/http_application_update/src/main.c

```diff
@@ -204,8 +254,11 @@ void main(void)
 		break;
 	}
 	printk("Initialized bsdlib\n");
-
-	modem_configure();
+	at_interface_init();
+#ifdef CONFIG_DFU_TARGET_MODEM
+	dfu_target_modem_delete_banked_fw(false, dfu_target_callback_handler);
+#endif
```

Comment:
I think this sample is doing application updates, not modem updates; why are we deleting the modem firmware here?

Reply:
This sample does both modem and application update as it's the default for `dfu_target` https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/samples/nrf9160/http_application_update/prj.conf

The reasoning is that the sample shouldn't break if you disable `DFU_TARGET_MODEM` but if we think it's fine that the sample breaks if you turn off a configuration I'm fine with removing the `#ifdef`